manifests/freshclam.pp

```diff
@@ -26,6 +26,17 @@
     content => template("${module_name}/clamav.conf.erb"),
   }
 
+  # NOTE: RedHat installations of clamav < 0.102 come with this flag which has been
+  # deprecated fro some time. When uprading to 0.102 or newer the setting perists,
+  # generating noise with cron jobs
+  file_line { 'Remove AllowSupplementaryGroups':
```

Comment:
This file is managed above. Doing this could result in puppet fighting itself.

This is not a default option here: https://github.com/edestecd/puppet-clamav/blob/master/manifests/params.pp#L252 

So i'm not sure how this is getting set for you. Puppet should mange that file content entirely resulting in any option not specified to be removed already.  Are you passing this option to the module?